### DISCUSSION TITLE
tailwind css is not run watch

### DISCUSSION THREAD

**Discussion by rony-1234** [tailwindlabs/tailwindcss#10492] [Help]
I Saw Tailwind css Terminal CIL and wrote the exact in Terminal.But I could not run the code .What is the reason?
![redircet](https://user-images.githubusercontent.com/89647309/216826521-407eb90b-e563-43ca-969c-710203103a4b.jpg)

**Comment by wongjn** [comment]
The CLI seems to be working correctly. As the warning message states, try using a Tailwind class or ensure that the `content` file globs match the files that contain Tailwind classes.
